File: tugas/python/Processor.py
class Processor():
    # constructor
    def __init__(self, name="-", price=0):
        self.__name = name
        self.__price = price

    # setter n getter
    def setName(self, name):
        self.__name = name

    def getName(self):
        return self.__name

    def setPrice(self, price):
        self.__price = price

    def getPrice(self):
        return self.__price

File: tugas/cpp/Processor.cpp
#include <bits/stdc++.h>

using namespace std;

class Processor{
    private:
        // atribut
        string name;
        int price;

    public:
        // constructor
        Processor(){
            this->name = "-";
            this->price = 0;
        }

        Processor(string name, int price){
            this->name = name;
            this->price = price;
        }

        // setter n getter
        void setName(string name){
            this->name = name;
        }

        string getName(){
            return this->name;
        }

        void setPrice(int price){
            this->price = price;
        }

        int getPrice(){
            return this->price;
        }

        ~Processor(){};
};